frontend/src/components/Question/Question.js

```diff
@@ -81,6 +81,9 @@ const Question = ({
             )}
             <h3 className={classNames({title: emphasizeTitle})}>{question.question}</h3>
             <div className="question">{render(question.view)}</div>
+            {question.expected_response &&
+                <p className="expected-response">{question.expected_response}</p>
+            }
```

Comment:
It would be good to add a comment here that `p.expected-response` will only ever be visible when the backend settings has `TESTING=True`.